src/guide/installation.md

```diff
@@ -12,6 +12,7 @@ You can still use an older Infection version if you're using an older PHP versio
 
 | PHP version | Infection version |
 |--|---|
+| 8.1.0 | >= 0.27.0 |
```

Comment:
IMO should be 0.26.16, as we did with PHP 8.0.0 in 0.26.7

in 0.* composer treats minor version as major, so I think we just need to continue 0.26.* branch here, as no BC changes nor new features have been added

WDYT?

Reply:
Yes there is no constraint semver wise, I was just thinking it might be a bit clearer this way. I don't feel strongly about it so entirely up to you.